src/compiler/binder.ts

```diff
@@ -313,7 +313,7 @@ import {
     unusedLabelIsError,
     VariableDeclaration,
     WhileStatement,
-    WithStatement,
+    WithStatement

```

Comment:
Organize Imports 💀

Reply:
Are you saying it removed the comma?

Reply:
(dprint when?)

Reply:
ASAP please. I need something to auto-fix everything when I’m flailing around with vim